ArjunaJTS/interop/README.md

```diff
@@ -0,0 +1,430 @@
+OVERVIEW
+--------
+
+This example shows how to propagate a transaction context between the WildFly and GlassFish application
+servers. There is a single script to run the full example but you may find it more instructive to run
+the example in steps perhaps in separate command windows:
```

Comment:
wouldn't be good to describe here a little bit why particular steps are needed?

particularly I mean - if I understand correctly to this quickstart works I need to take sourcecode of WFLY and GlassFish, patch it and then run the test. I think this point would be good to be stress out and explained why it's so and/or if this work in future without such patching.